MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3771,7 +3771,7 @@ public string GetLibplaceboFilter(
                 // sw => hw
                 if (doOclTonemap)
                 {
-                    mainFilters.Add("hwupload=derive_device=d3d11va:extra_hw_frames=16");
+                    mainFilters.Add("hwupload=derive_device=d3d11va:extra_hw_frames=24");
```

Comment:
You mentioned changing the order but this seems to change the frame pool size to be allocated, 24 still seems to be very conservative with some of the QSV examples I found. But not sure this was intentional or this happened to slip into this pr.

Reply:
nvm, re-reading the commit messages it also mentioned adapting fro ffmpeg 7 separately from changing the order, so this is probably intended.